tests/integration/authentication.test.ts

```diff
@@ -14,12 +14,18 @@ async function bootstrapConnectModalPageTest(demo: DemoPageObject, auth: AuthPag
   const authPage = await newWindow.page();
   expect(authPage.url().startsWith(auth.url)).toBeTruthy();
   await authPage.waitFor(auth.$inputUsername, { timeout: 15000 });
-  authPage.on('console', event => {
-    console.log(event.type(), event.text());
-  });
+  // authPage.on('console', event => {
+  //   console.log(event.type(), event.text());
+  // });
```

Comment:
nit: this should be removed, not commented out

Reply:
:+1: fixed